src/Flysystem/FlysystemAssetStore.php

```diff
@@ -1233,7 +1234,15 @@ public function getMetadata($filename, $hash, $variant = null)
         // If `applyToFileOnFilesystem` calls our closure we'll know for sure that a file exists
         return $this->applyToFileOnFilesystem(
             function (ParsedFileID $parsedFileID, Filesystem $fs) {
-                return $fs->getMetadata($parsedFileID->getFileID());
+                $path = $parsedFileID->getFileID();
+                return [
+                    'timestamp' => $fs->lastModified($path),
```

Comment:
We should be using `$fs->getAdapter()->getMetaData()`? similar to 1.x? https://github.com/thephpleague/flysystem/blob/1.x/src/Filesystem.php#L357

Reply:
`getMetadata` method doesn't exist anymore.
Currently, it was replaced with few new method for each metadata attribute.
[Filesystem::getMetadata is removed](https://flysystem.thephpleague.com/docs/upgrade-from-1.x/)
[Metadata getters are renamed](https://flysystem.thephpleague.com/docs/upgrade-from-1.x/)